products/idcard_fr/response_v2/default_sample.rst

```diff
@@ -0,0 +1,53 @@
+########
+Document
+########
+:Mindee ID: d33828f1-ef7e-4984-b9df-a2bfaa38a78d
+:Filename: default_sample.jpg
+
+Inference
+#########
+:Product: mindee/idcard_fr v2.0
+:Rotation applied: Yes
+
+Prediction
+==========
+:Nationality:
+:Card Access Number: 175775H55790
+:Document Number:
+:Given Name(s): Victor
+              Marie
```

Comment:
```suggestion
:Given Name(s): Victor
                Marie
```

Reply:
I can't tell whether it's me or if GitHub is still bugging like yesterday, but that change has been made already :thinking: